luigi_pipeline/lib/model/gcnv_mt_schema.py

```diff
@@ -173,7 +173,7 @@ def samples_qs(self, start=0, end=1000, step=10):
     @row_annotation(name="samples_cn", fn_require=SeqrGenotypesSchema.genotypes)
     def samples_cn(self, start=0, end=4, step=1):
         return hl.struct(**{
-            '%i' % i: self._genotype_filter_samples(lambda g: g.cn == i)
+            f'{i}_to_{i + step}': self._genotype_filter_samples(lambda g: g.cn == i)
```

Comment:
this should just be `f'{i}'` I think

Reply:
this is what I get for not running the tests :(. This is even covered!